### PR TITLE
"Easier life with XCode marks while using Bash scripts" hint

### PR DESCRIPTION
**Pull request by kondratk**

### CODE REVIEW

**Comment by wojteklukaszuk** [comment]
Change title a little. It is too long right now.

File: source/_posts/2015-07-23-easier-life-with-xcode-marks-while-using-bash-scripts.markdown [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "Easier life with XCode marks while using Bash scripts"
```
Comment (wojteklu) [inline source/_posts/2015-07-23-easier-life-with-xcode-marks-while-using-bash-scripts.markdown]: Change title a little. It is too long right now.

File: source/_posts/2015-07-23-easier-life-with-xcode-marks-while-using-bash-scripts.markdown [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+date: 2015-07-23 13:24:06 +0200
+comments: false
+author: kondrat
+categories:
+---
+Bash scripts are commonly used in Mac/ iOS development to make some repeatable operations hands-free, however sometimes things go wrong, and in perfect world I’d like to be properly informed about the errors.
```
Comment (wojteklu) [inline source/_posts/2015-07-23-easier-life-with-xcode-marks-while-using-bash-scripts.markdown]: remove space in Mac/ iOS.

Comment (wojteklu) [inline source/_posts/2015-07-23-easier-life-with-xcode-marks-while-using-bash-scripts.markdown]: try to limit long sentences. Start new sentence with 'however'.

File: source/_posts/2015-07-23-easier-life-with-xcode-marks-while-using-bash-scripts.markdown [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Here comes view called „Report Navigator”, which displays any type of errors, warnings at compile time of Objective-C/ Swift, or even while typing new lines of code. Let me show you how to push warnings, and errors that come out of Bash script into the Report Navigator:
```
Comment (wojteklu) [inline source/_posts/2015-07-23-easier-life-with-xcode-marks-while-using-bash-scripts.markdown]: Remove space after '/'.

File: source/_posts/2015-07-23-easier-life-with-xcode-marks-while-using-bash-scripts.markdown [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+else
+    echo "warning: running on Debug target"
+fi
+```
+
+Please, take a look in every echo at `note:`, `warning:` and `error:` prefixes. When XCode recognizes any of these tags, the proper indicator will show up in Report Navigator, and of course at the top bar:
+
+{% img center /images/bash-scripts-marks/bash-scripts-error-navigator.png %}  
+
+Pay attention to the colon at the end of each tag. Also the tags don’t have to be prefixes of each echo message- feel free to place it anywhere at the string.
```
Comment (wojteklu) [inline source/_posts/2015-07-23-easier-life-with-xcode-marks-while-using-bash-scripts.markdown]: "Also the tags don’t have to be prefixes of each echo message- feel free to place it anywhere at the string." - change to something more readable

File: source/_posts/2015-07-23-easier-life-with-xcode-marks-while-using-bash-scripts.markdown [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+if [ "Debug"= "${CONFIGURATION}" ];
+then
+    echo "error: please run this script on Release target"
+else
+    echo "warning: running on Debug target"
+fi
+```
+
+Please, take a look in every echo at `note:`, `warning:` and `error:` prefixes. When XCode recognizes any of these tags, the proper indicator will show up in Report Navigator, and of course at the top bar:
```
Comment (wojteklu) [inline source/_posts/2015-07-23-easier-life-with-xcode-marks-while-using-bash-scripts.markdown]: remove ',' before 'and'

File: source/_posts/2015-07-23-easier-life-with-xcode-marks-while-using-bash-scripts.markdown [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+categories:
+---
+Bash scripts are commonly used in Mac/ iOS development to make some repeatable operations hands-free, however sometimes things go wrong, and in perfect world I’d like to be properly informed about the errors.
+Here comes view called „Report Navigator”, which displays any type of errors, warnings at compile time of Objective-C/ Swift, or even while typing new lines of code. Let me show you how to push warnings, and errors that come out of Bash script into the Report Navigator:
+
+As usual create new `Run Script Phase`, and insert your Bash script:
+
+{% img center /images/bash-scripts-marks/bash-scripts-new-script-phase.png %}  
+
+For blog post purposes I’ve created simple script that checks which configuration is used for build:
```
Comment (wojteklu) [inline source/_posts/2015-07-23-easier-life-with-xcode-marks-while-using-bash-scripts.markdown]: for the build*

File: source/_posts/2015-07-23-easier-life-with-xcode-marks-while-using-bash-scripts.markdown [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+title: "Easier life with XCode marks while using Bash scripts"
+date: 2015-07-23 13:24:06 +0200
+comments: false
+author: kondrat
+categories:
+---
+Bash scripts are commonly used in Mac/ iOS development to make some repeatable operations hands-free, however sometimes things go wrong, and in perfect world I’d like to be properly informed about the errors.
+Here comes view called „Report Navigator”, which displays any type of errors, warnings at compile time of Objective-C/ Swift, or even while typing new lines of code. Let me show you how to push warnings, and errors that come out of Bash script into the Report Navigator:
+
+As usual create new `Run Script Phase`, and insert your Bash script:
```
Comment (wojteklu) [inline source/_posts/2015-07-23-easier-life-with-xcode-marks-while-using-bash-scripts.markdown]: remove ',' before and

**Pull request merged**